client/src/js/directives/bhLocationSelect.js

```diff
@@ -77,32 +78,29 @@ function LocationSelectController(Locations, $scope) {
 
   /**
    * <select> component messages to be translated
-   * if there is no data, indicate that to the user.
    */
-  var selectCountry  = 'SELECT.COUNTRY';
-  var selectProvince = 'SELECT.PROVINCE';
-  var selectSector   = 'SELECT.SECTOR';
-  var selectVillage  = 'SELECT.VILLAGE';
-  var noData         = 'SELECT.EMPTY';
   vm.messages = {
-    country:  selectCountry,
-    province: selectVillage,
-    sector:   selectSector,
-    village:  selectVillage,
+    country:  Locations.messages.country,
+    province: Locations.messages.province,
+    sector:   Locations.messages.sector,
+    village:  Locations.messages.village,
   };
 
-  // load the countries once, at startup
-  Locations.countries()
-  .then(function (countries) {
-    vm.countries = countries;
-
-    // if there are countries to select, show a "select a country" message
-    // however, if there isn't any data, show a "no data" message. This pattern
-    // is used throughout the component.
-    vm.messages.country = (countries.length > 0) ?
-      selectCountry :
-      noData;
-  });
+  function loadCountries() {
+    Locations.countries()
```

Comment:
This API was introduced in a previous pull request that created the LocationService.  I was using the same syntax as the `PatientService`, which allows the developer to do `Patients.groups()` to get the groups associated with a patient.  [See here](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/js/services/patients.js#L19).

I'm happy to update particular query if it seems better.  Curious about what @sfount thinks.